ner_v2/detectors/numeral/number/zh-TW/data/units.csv

```diff
@@ -0,0 +1,2 @@
+unit_type,unit_value,unit_variants
+currency,dollar,Dollar | usd | डॉलर | $
```

Comment:
@harjinder7 ,  what are the values generally placed in units.csv and shouldn't we be adding Chinese currency too, please check the other languages for consistency and add accordingly or place a TODO if it takes more effort.

Reply:
this file generally have things like the currency, package_metric_unit like gram, kilograms, mililitre etc
languages other than english they have only currency as unit in this file